requestor/requirements.txt

```diff
@@ -2,4 +2,4 @@ Quart==0.15.1
 quart-cors==0.5.0
 web3==5.20.0
 
-git+https://github.com/golemfactory/yapapi-service-manager.git
+git+https://github.com/golemfactory/yapapi-service-manager.git@wiezzel/linters-APPS-188
```

Comment:
To be removed when https://github.com/golemfactory/yapapi-service-manager/pull/9/ is merged.